tools/cldr-apps/js/src/esm/cldrLoad.js

```diff
@@ -620,6 +620,20 @@ function unspecialLoad(itemLoadInfo, theDiv) {
     }
   } else if (curSpecial) {
     console.log("No special js found for " + curSpecial);
+    cldrSurvey.hideLoader();
+    isLoading = false;
+    flipper.flipTo(pages.other, document.createTextNode("Page Not Found")); // stop loader
+    notification.warning({
+      message: "Page not found",
+      description: `The page “${curSpecial}” does not exist.
+        The link could be out-of-date, or you may have found a bug.
+        Click here to return to the SurveyTool.`,
+      duration: 0,
```

Comment:
I'd rather see an actual button to click on rather than making the whole notification act like a button; this would enable selecting and copying the text